### ISSUE TITLE
[Homepage] Keyboard tabbing to various buttons + links doesn't work in Safari

### ISSUE THREAD

**Issue by mollieru** [18F/web-design-standards#539]
*Steps to recreate*
1. option+tab around the Homepage
2. These buttons don't seem to have their focus state's/ability to click appear:
> 'download the components' 

> *Tools for Creating Beautiful...' section* ''Visual Style Guide' link

> links to open each of the 'examples'

NOTE: This may be *just* because the links don't yet have anything they're pointed to yet. If that's not the case, and this is an actual bug, I will add a video demonstrating. 

*Tested in*
OS 10.8
Safari 6.2

**Comment by carodew** [comment]
Possibly related to #525

**Comment by msecret** [comment]
So if this is only because these links are pointing to anything, I'm guessing we should ignore this for now because they will eventually point to something?

**Comment by carodew** [comment]
The download the components buttons seems to have a focus state, it just appears faintly against the dark blue background:

<img width="341" alt="screen shot 2015-09-17 at 1 58 25 pm" src="https://cloud.githubusercontent.com/assets/10144074/9941584/44a55132-5d44-11e5-9ff8-dbe58d9875f2.png">

This was discussed in #537 and decided to be an enhancement. 

The rest of the links actually point to things so that is not the issue. But I believe they are covered in #526 and #528. Sorry I didn't catch that earlier. 

With that, it seems like everything in this issue is already covered elsewhere, so I'm going to go ahead and close it.
